TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -86,6 +86,11 @@ public bool UseEuroInformationLegacyReplacingSyntax
         /// </summary>
         public TypeCobolCheckOption CheckRecursivePerforms { get; set; }
 
+        /// <summary>
+        /// Use to set that these options are used by some tests.
+        /// </summary>
+        public bool IsTestMode { get; set; }
```

Comment:
So in some tests, it appears that the `Debug.Assert` in CrossChecker fails, but we cannot add such property in `TypeCobolOptions`. It has no meaning outside of the project.

Reply:
So shall we remove this Debug Assert?

Reply:
Assert removed in 6a515c8